seed/lib/mcm/reader.py

```diff
@@ -405,40 +428,23 @@ class CSVParser(object):
             # rows.next() will return the first row
     """
 
-    def __init__(self, csvfile, *args, **kwargs):
```

Comment:
I found no instances of *args or **kwargs being used for initialization, so I removed them (and their downstream uses)